pkg/shell/cockpit-server.js

```diff
@@ -72,6 +72,13 @@ function ServerTime() {
         }
```

Comment:
Lets call the 'now' property 'utc_fake_now' and call out its strange behavior in its name.